rust/src/dcerpc/dcerpc.rs

```diff
@@ -1183,9 +1188,10 @@ pub extern "C" fn rs_parse_dcerpc_response_gap(
 
 #[no_mangle]
 pub extern "C" fn rs_dcerpc_parse_request(
-    flow: *mut core::Flow, state: &mut DCERPCState, _pstate: *mut std::os::raw::c_void,
-    input: *const u8, input_len: u32, _data: *mut std::os::raw::c_void, flags: u8,
+    flow: *const core::Flow, state: *mut std::os::raw::c_void, _pstate: *mut std::os::raw::c_void,
```

Comment:
This cannot go as a separate commit as these are the signature changes required by the "register" function in Rust.
Bringing them out as separate commits would break the build.

Reply:
Cannot these be done before using the register function ?

Reply:
I tried it and they can be done. Please check in the new PR.